portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -61,9 +61,11 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       }
       /* Convert its message to the requested langauge */
       Translation translation =
-        translate.translate((String)entity.getProperty("text"), 
-        Translate.TranslateOption.targetLanguage(lang));
+          translate.translate((String)entity.getProperty("text"), 
```

Comment:
I'm having trouble finding the code where the timestamp is translated. Maybe I'm reading these lines improperly, but it seems like you're only translating the message part of the comment here. A quick comment might help the reader.